Libraries/Lists/FlatList.js

```diff
@@ -126,7 +126,13 @@ type OptionalProps<ItemT> = {|
    * Multiple columns can only be rendered with `horizontal={false}` and will zig-zag like a
    * `flexWrap` layout. Items should all be the same height - masonry layouts are not supported.
    */
-  numColumns: number,
+  numColumns?: number,
+  /**
+   * Note: may have bugs (missing content) in some circumstances - use at your own risk.
+   *
+   * This may improve scroll performance for large lists.
```

Comment:
Same here, adding a comment that default is true for Android

Reply:
Like above - a suggestion:
```suggestion
   * This may improve scroll performance for large lists.
   *
   * The default value is true.
```
What do you think?